### PR TITLE
DOCSP-37640 Document loggedStages

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-37640
Staging - https://preview-mongodbjordansmith721.gatsbyjs.io/csharp/DOCSP-37640-loggedStages/fundamentals/linq/#view-translated-queries

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/linq.txt [hunk 1]
```diff
@@ -575,3 +582,75 @@ implementation of LINQ:
 
    * - ``Where``
      - Returns all documents that match your specified criteria
+
+View Translated Queries
+-----------------------
+
+When you run a LINQ query, the {+driver-short+} automatically translates your
+query into an aggregation pipeline written with the {+query-api+}. You can view
```
Comment (mcmorisi) [inline source/fundamentals/linq.txt:590]: This currently reads "written with the MongoDB Query API" but the paragraph refers to translation, which to me implies another language. Would this be better as "written in the MongoDB Query Language (MQL)"?

File: source/fundamentals/linq.txt [hunk 2]
```diff
@@ -575,3 +582,75 @@ implementation of LINQ:
...
+
+   .. input::
+      :language: csharp
+
+       var queryableCollection = _restaurantsCollection.AsQueryable();
+       var query = queryableCollection
+             .Where(r => r.Name == "The Movable Feast");
+
+         var queryTranslated = query.ToString();
+         Console.WriteLine(queryTranslated);
```
Comment (mcmorisi) [inline source/fundamentals/linq.txt:610]: Indentation

```suggestion
       var queryTranslated = query.ToString();
       Console.WriteLine(queryTranslated);
```

**Review by mcmorisi — CHANGES_REQUESTED** [review]
Good job! Just a few points to address.

File: source/fundamentals/linq.txt (hunk 1 above)
Comment (jordan-smith721) [reply source/fundamentals/linq.txt:590]: We don't refer to MQL as MQL or MongoDB Query Language externally anymore. It's been changed to the MongoDB Query API

**Review by mcmorisi — APPROVED** [review]
LGTM!

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request merged**